scripts/templates/EmptyDemo.mustache

```diff
@@ -0,0 +1,92 @@
+/* tslint:disable:no-unused-variable */
+import * as React from 'react';
+/* tslint:enable:no-unused-variable */
+import * as ReactDOM from 'react-dom';
+import { App, AppDefinition } from './AppDefinition';
+import { IAppLink, IAppLinkGroup } from '@uifabric/example-app-base';
+import { Router, Route } from 'office-ui-fabric-react/lib/utilities/router/index';
+import { GettingStartedPage } from './GettingStartedPage';
+import { setBaseUrl } from 'office-ui-fabric-react/lib/Utilities';
+import { Fabric } from 'office-ui-fabric-react/lib/Fabric';
+import { initializeIcons } from 'office-ui-fabric-react/lib/Icons';
```

Comment:
Also possibly affected by package-level import "standard"

Reply:
Most of these imports aren't exported from office-ui-fabric-react index, so they actually need to be package-level imports here.